.github/workflows/label-notifications.yml

```diff
@@ -34,7 +35,7 @@ jobs:
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": ":warning: The following has been labeled as a regression:"
+                    "text": ":warning: The following has been labeled as a $LABEL_NAME:"
```

Comment:
I'm fairly confident that `$LABEL_NAME` would work here too, but using `${{ env.LABEL_NAME }}` keeps consistency with line 45 below.

```suggestion
                    "text": ":warning: The following has been labeled as a ${{ env.LABEL_NAME }}:"
```

Reply:
Since it's only referenced once, I like using the initial value directly. I'll get this updated.

Reply:
Fixed with [6ec8dea](https://github.com/hashicorp/terraform-provider-aws/pull/39015/commits/6ec8dea4f2126894a44a785bbc58dd8de9b6ec5d)